lib/core/src/Cardano/Wallet.hs

```diff
@@ -1458,7 +1467,6 @@ instance Buildable PartialTx where
 balanceTransaction
     :: forall m s k ctx.
         ( HasTransactionLayer k ctx
-        , HasLogger m WalletWorkerLog ctx
```

Comment:
Should probably preserve the logging in the new `selectAssets'`

Reply:
Should be easily doable to preserve: https://github.com/input-output-hk/cardano-wallet/blob/70cc0619e7c23def2747ab602ef5602741a61076/lib/core/src/Cardano/Wallet.hs#L2144-L2149

Possibly some new log message could be added as well… Should look at it…

Reply:
I added a `MsgSelectionForBalancingStart UTxO PartialTx` analogous to `MsgSelectionStart UTxO [TxOut]`, and kept the SelectionError/Report ones the same.